tests/tests/features-kubernetes.yaml

```
@@ -26,6 +26,17 @@
     project: ci-features-{{ lookup('env','CLUSTER_TYPE') }}
     branch: ingress-annotations
 
+- include: features/short-router-url.yaml
+  vars:
+    testname: "SHORT ROUTER URL {{ cluster_type|upper }}"
+    git_repo_name: features.git
+    project: ci-features-{{ cluster_type }}
```

```suggestion
    project: ci-features-{{ lookup('env','CLUSTER_TYPE') }}
```
Variable differences between 1.x and 2.x